src-docs/src/views/inline_edit/inline_edit_example.js

```diff
@@ -82,5 +86,25 @@ export const InlineEditExample = {
       demo: <InlineEditConfirm />,
       props: { EuiInlineEditText },
     },
+    {
+      title: 'Loading state',
```

Comment:
I'd like to see us cover both loading and invalid states in the same section if possible!
```suggestion
      title: 'Loading and invalid states',
```